viz.yaml

```diff
@@ -246,6 +246,8 @@ publish:
       map_states: map_states
       map_counties: "map_counties"
       dropdown_selector: "dropdown_selector"
+      national_pie: "national_pie"
+      rank_states_utils: "rank_states_utils"
```

Comment:
Should we delete these dependencies from https://github.com/USGS-VIZLAB/water-use-15/pull/286/files#diff-181bf6189cfb48aabd24db45de8426a8R397 now that they're called directly in build_map.js?

Reply:
Oh, good point. I'll get a PR up, but I'm going to sign off for tonight and say that we are in a good place for reviewers right now.